### DISCUSSION TITLE
Generative AI fix on G search

### DISCUSSION THREAD

**Discussion by noosemanAL** [uBlockOrigin/uAssets#23710] [Ideas]
Some elements, You have to agree to AI terms to even disable according to their tutorial, Not acceptable, We need a full block to prevent data collection if possible. I disabled via chrome-labs under chrome://flags but didn't hide.

```adb
www.google.com##.h7Tj7e
www.google.com##.M8OgIe
www.google.com###_9s5EZsrNAr6NwbkPov2bkAI_27
www.google.com##.ssOUyb.KxwPGc > .pHiOh
www.google.com##a.pHiOh:nth-of-type(3)
www.google.com##.AghGtd.KxwPGc
www.google.com##a.MV3Tnb:nth-of-type(2)
www.google.com##a.MV3Tnb:nth-of-type(1)
www.google.com##.Fa3Wfb > path
www.google.com##strong
www.google.com##.NOo3C > span > span
www.google.com##.PZPZlf.MNX06c.uP58nb.rPeykc
www.google.com##.vM0jzc > span
www.google.com##.B4fN8b.xFTqob
www.google.com###dimg_stZEZrn_HY-Zwt0Phpue6AQ_1
www.google.com###dimg_stZEZrn_HY-Zwt0Phpue6AQ_2
www.google.com##.Fgizfe.z1asCe.okxdqe > svg
www.google.com##.Fgizfe.z1asCe.okxdqe
www.google.com##.PZPZlf.GmFi7
www.google.com##.PZPZlf.rPeykc > span
www.google.com##.dPgQ4d.rfV7Hf.Gz05vf.DuQANe
www.google.com##.gzV1t.rlt7Ub.FR7ZSc.NQYJvc.fCrZyc.k0Jjg.btku5b.oQOpie.in7vHe > .SlP8xc.VDgVie.niO4u
www.google.com##.Jzkafd.zNsLfb
www.google.com##.Jzkafd.RDmXvc
www.google.com##.dSKvsb.zMgcWd
www.google.com##.LT6XE
www.google.com##.rNSxBe.Pqkn2e
www.google.com##.RK5Kkf.GkDqAd.WAUd4.OZ9ddf
www.google.com##.M8OgIe
```

**Comment by krystian3w** [comment]
The filter is definitely of poor quality:

> ```adb
> www.google.com##strong
> ```

Assuming that these elements do not load ads and bare HTML5 does not track clicks, you should create your own list and not throw it into: EL, EP, uBo - Ads or uBo - Privacy.

**Comment by noosemanAL** [comment]
"[noosemanAL](https://github.com/noosemanAL) asked this question in [Ideas]"
This is an "idea" not an issue tracker or filter, The Idea is there needs to be a competent filter to block the AI component in Google Web search. regardless the browser. The AI will steal data, Way more than cookies ever did.
This is not a filter, these are just some elements I personally blocked from the AI on search results, I am not experienced in ublock enough to create the AI block.

**Reply by krystian3w** [reply]
But `##strong` hide any bold element at Google.

**Reply by krystian3w** [reply]
https://www.w3schools.com/tags/tag_strong.asp

**Comment by noosemanAL** [comment]
Thank You, I never mentioned any **BOLD** element, I mentioned the new 'Generative AI' Search Module that appears when making a search using "G search" is made. 
![AI](https://github.com/uBlockOrigin/uAssets/assets/52615094/6a61f536-bb9f-4a6a-af57-59d5a4826087)
(I removed the actual logo, and put in a placeholder logo of my own creation.)

**Comment by krystian3w** [comment]
Maybe work as long exists `udm=14`:

```adb
!#if ext_ublock
||google.*/search?*q=$uritransform=/$/&udm=14/
@@||google.*/search?*q=*udm=2$uritransform=/$/&udm=14/
@@||google.*/search?*q=*udm=14$uritransform=/$/&udm=14/
@@||google.*/search?*q=*tmb=$uritransform=/$/&udm=14/
!#if env_mobile
!#if env_firefox
@@||google.*/search?*q=$uritransform=/$/&udm=14/
!#endif
!#endif
!#endif
```

use at own risk.
